### PR TITLE
Update MongoDB versions, target on Rails 5.0.0.rc1

### PR DESCRIPTION
**Pull request by Fudoshiki**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,23 +6,22 @@ bundler_args: --without development
...
 rvm:
   - 2.2.2
-  - 2.3.0
+  - 2.3.1
   - jruby-9.0.5.0
 
 gemfile:
   - Gemfile
   - gemfiles/rails5master.gemfile
-  - gemfiles/rails5rc1.gemfile
```
Comment (estolfo) [inline .travis.yml]: Thanks for doing these updates, but why did you delete the rails5rc1 gemfile? I'd like to confirm in travis that our tests pass on rc1. If you could add it back in, that'd be great.

Also, we can update jruby to jruby-9.1.1.0 in travis.yml now as well.

Thanks!

Comment (Fudoshiki) [inline .travis.yml]: @estolfo Can You cancel travis builds except last commit?
